src/pages/landing-pages/[landingPage].tsx

```diff
@@ -0,0 +1,138 @@
+import Bookmark from 'components/Bookmark/Bookmark'
+import {
+  DocumentType,
+  DocumentSectionType,
+  Collection as CollectionType,
+} from 'types'
+import { isPdf, handleOpenPdfLink } from 'helpers/openDocumentLink'
+import styles from 'styles/pages/landingPages.module.scss'
+
+const LandingPage = ({
+  pageTitle,
+  pageDescription,
+  articles,
+  collections,
+  documents,
+}: InferGetServerSidePropsType<typeof getServerSideProps>) => {
+  const { user } = useUser()
+
+  return (
+    <GridContainer>
+      {!user ? (
+        <Loader />
+      ) : (
+        <div>
+          <h1>{pageTitle}</h1>
+          <p>{pageDescription}</p>
+          <h2>Documentation</h2>
+          <Grid row gap="lg">
+            {documents &&
+              documents.map((s: DocumentSectionType, i: number) => (
+                <Grid col={12} key={s.id} className={styles.accordionGrid}>
+                  <Accordion
+                    className={styles.accordion}
+                    bordered
+                    items={[
+                      {
+                        title: s.title,
+                        expanded: i === 0 ? true : false,
+                        id: `${s.id}`,
+                        headingLevel: 'h3',
+
+                        content: (
+                          <div className={styles.accordionContent}>
+                            {s.document.map((d: DocumentType) => (
+                              <Link
+                                onClick={(e) => {
+                                  if (isPdf(d.file.url)) {
+                                    e.preventDefault()
+                                    handleOpenPdfLink(d.file.url)
+                                  } else return
+                                }}
+                                key={d.id}
+                                rel="noreferrer noopener"
+                                href={`${d.file.url}`}>
+                                {d.title}
+                              </Link>
+                            ))}
+                          </div>
+                        ),
+                      },
+                    ]}
+                  />
+                </Grid>
+              ))}
+          </Grid>
+          <h2>Articles</h2>
+          <ArticleList articles={articles} />
+          <h2> Collections</h2>
+          <Grid row gap>
+            {collections.map((collection: CollectionType) => (
+              <Collection
+                key={`collection_${collection.id}`}
+                title={collection.title}
+                className={'sitesAndAppsCollection'}>
+                {/* TODO Fix type here */}
+                {/* eslint-disable-next-line @typescript-eslint/no-explicit-any */}
+                {collection.bookmarks?.map((bookmark: any) => (
+                  <Bookmark
+                    className={'sitesAndAppsBookmark'}
+                    key={`bookmark_${bookmark.id}`}
```

Comment:
Just wanted to flag for later that both the Collection key and the Bookmark key are coming back undefined and printing to the console as a duplicate key error.